projects/plugins/protect/src/js/routes/scan/index.jsx

```diff
@@ -77,7 +77,7 @@ const ScanPage = () => {
 					>
 						<Col>
 							<div ref={ setScanResultsAnchor }>
-								<ScanResultsDataView filters={ filters } />
+								<ScanResultsDataView filters={ hasPlan ? filters : null } />
```

Comment:
Nit - this `hasPlan` check could be included inside the `const filters = useMemo( ... )` definition.